apps/integration-ui/src/globals.css

```
@@ -1,5 +1,27 @@
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss/utilities' layer(utilities);
+
+@plugin 'tailwindcss-animate';
+
+@source '../../../packages/widgets/dist/**/*.js';
+
+@custom-variant dark (&:is(.dark *));
+
+@theme {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)